htdocs/manage/settings/index.bml

```diff
@@ -32,8 +32,16 @@ body<=
     my $authas;
     my $u;
 
-    if ($remote) {
-        $authas = $GET{authas} || $remote->user;
+    $authas = $GET{authas} || $remote->user if $remote;
+
+    my $can_view_other = ( $remote && $remote->has_priv( "canview", "subscriptions" ) );
+
+    if ( $can_view_other && $GET{cat} && $GET{cat} eq 'notifications' ) {
+        # impersonation mode - check for $u->user ne $authas
+        $u = LJ::load_user( $GET{user} ) if $GET{user};
```

Comment:
It could be - it's syntactically the same either way, and I thought it was visually clearer to have the two $GET{user} references adjacent to each other.